python_bindings/src/PyEnums.cpp

```diff
@@ -82,6 +82,21 @@ void define_enums(py::module &m) {
         .value("RISCV", Target::Arch::RISCV)
         .value("WebAssembly", Target::Arch::WebAssembly);
 
+    py::enum_<Target::Processor>(m, "TargetProcessor")
+        .value("TuneGeneric", Target::Processor::ProcessorGeneric)
```

Comment:
Suggestion: here and elsewhere, consider always listing all these enums in lexicographical order, so that there is an obvious and consistent order and thus additions will go into an obvious place.